src/server/config.ini

```diff
@@ -280,6 +280,11 @@
   # Bloom filter type, should be either 'common' or 'prefix'
   rocksdb_filter_type = prefix
 
+  # 10MB, 1000, 30s
+  rocksdb_multi_get_max_iteration_size = 10000000
```

Comment:
10 << 20

Reply:
I update to `rocksdb_multi_get_max_iteration_size = 10485760` here. It seems that `10 << 20` is an invalid value while parsing options from config.

Reply:
What is invalid mean? parse failure?

Reply:
We read the value as a string_view, and use a function called `buf2unsigned` to convert it into a uint_64 value. `10 << 20` is not a valid string which can be parsed into `uint64_t`.